aten/src/THCUNN/generic/SpatialConvolutionMM.cu

```diff
@@ -348,38 +365,41 @@ void THNN_(SpatialConvolutionMM_accGradParameters)(
            accreal scale_) {
 
   real scale = ScalarConvert<accreal, real>::to(scale_);
-  THCUNN_assertSameGPU(state, 5, input, gradOutput, gradWeight, columns, ones);
-  if (gradBias) {
-   THCUNN_assertSameGPU(state, 2, gradWeight, gradBias);
+  THCUNN_assertSameGPU(state, 5, input, gradOutput, gradWeight, gradBias, columns, ones);
+  if (gradWeight) {
+    THArgCheck(THCTensor_(isContiguous)(state, gradWeight), 4, "gradWeight needs to be contiguous");
   }
```

Comment:
This should probably call newContiguous. In general people can’t control if grads given to their functions will be contiguous or not

Reply:
I'll change all `gradWeight` to use `newContiguous` :). 

Also, there are `isContiguous` vs `newContiguous` inconsistencies across the files. I'm not changing any of the existing checks (`newContiguous` is still `newContiguous` and `isContiguous` is still `isContiguous`) except moving them around a little bit.

Reply:
Well, upon thinking about it. `gradWeight` should always be contiguous. It is the output of `accGradParameters`. Now in ATen where this function is called, a contiguous tensor is always constructed and then fed in as `gradWeight`.

Reply:
Ah sorry. I don't know why I thought `gradWeight` is the input. It's ok to have `isContiguous` then